### PR TITLE
Updated Logstash hook library and fixed its usage

### PR DESCRIPTION
**Pull request by vgarvardt**

### CODE REVIEW

**Comment by stefvaessen** [comment]
+1

[Approved with PullApprove](https://pullapprove.com/hellofresh/logging-go/pull-request/9/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=hellofresh/logging-go) [Approved with PullApprove](https://pullapprove.com/hellofresh/logging-go/pull-request/9/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=hellofresh/logging-go)

File: hooks.go [hunk 1]
```diff
@@ -22,9 +22,14 @@ func (c LogConfig) initLogstashHook(h LogHook) error {
 		return ErrFailedToConfigureLogHook
 	}
 
-	formatter := getLogstashFormatter(h.Settings)
+	formatter := getLogstashFormatter(h.Settings).(*logrustash.LogstashFormatter)
```
Comment (rafaeljesus) [inline hooks.go:25]: shouldn't we catch cast error here?

Comment (vgarvardt) [reply hooks.go:25]: no - check the implementation, and I have a test for this =)

**Comment by rafaeljesus** [comment]
👍 

[Approved with PullApprove](https://pullapprove.com/hellofresh/logging-go/pull-request/9/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=hellofresh/logging-go) [Approved with PullApprove](https://pullapprove.com/hellofresh/logging-go/pull-request/9/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=hellofresh/logging-go)

**Pull request merged**